### PR TITLE
Issue#1036 - Add new API that returned markers for a given polygon

### PR DESCRIPTION
**Pull request by erangil**
Issue#1036 - Add new API that returnד markers for a given polygon

### CODE REVIEW

File: anyway/accidents_around_schools.py [hunk 1]
```diff
@@ -169,4 +169,4 @@ def main(start_date, end_date, distance, output_path):
     joined_df.columns = [col if type(col)==str else '_'.join(map(str, col)) for col in joined_df.columns.values]
     joined_df = joined_df.loc[:,['school_yishuv_name', 'school_name', 'rank_in_yishuv', 'school_longitude', 'school_latitude', 'injured_count_1', 'injured_count_2', 'injured_count_3', 'total_injured_count_', 'anyway_link_with_filters', 'school_id']]
     joined_df.columns = ['school_yishuv_name', 'school_name', 'rank_in_yishuv', 'school_longitude', 'school_latitude', 'killed_count', 'severly_injured_count', 'light_injured_count', 'total_injured_killed_count', 'anyway_link', 'school_id']
-    joined_df.to_csv(os.path.join(output_path,'df_total_involved_count_by_yishuv.csv'), encoding=CONTENT_ENCODING, header=True)
+    joined_df.to_csv(os.path.join(output_path,'df_total_involved_count_by_yishuv.csv'), encoding=CONTENT_ENCODING, header=True)
```
Comment (atalyaalon) [inline anyway/accidents_around_schools.py]: @erangil  this file shouldn't be changed

File: anyway/flask_app.py [hunk 2]
```diff
@@ -1311,6 +1312,29 @@ def get_dict_file(directory):
         yield name, df
 
 
+@app.route("/markers/polygon/", methods=["GET"])
+def acc_in_area_query():
+    # pol_str will be received in the following format: 'POLYGON(({lon} {lat},{lon} {lat},........,{lonN},{latN}))'
+    # please note that start point and end point must be equal: i.e. lon=lonN, lat=latN
+    pol_str = request.values.get('polygon', "")
+    start_date = utilities.valid_date(request.values.get('start_date', '01-01-2013'))
+    end_date = utilities.valid_date(request.values.get('end_date', '31-12-2017'))
```
Comment (atalyaalon) [inline anyway/flask_app.py]: @erangil you can filter out dates, only if they come from the user, better not to use such default dates

File: anyway/flask_app.py [hunk 3]
```diff
@@ -1311,6 +1312,29 @@ def get_dict_file(directory):
...
+        .filter(AccidentMarker.geom.intersects(pol_str)) \
+        .filter(or_((AccidentMarker.provider_code == CONST.CBS_ACCIDENT_TYPE_1_CODE),
+               (AccidentMarker.provider_code == CONST.CBS_ACCIDENT_TYPE_3_CODE))) \
+        .filter(AccidentMarker.created >= start_date) \
+        .filter(AccidentMarker.created < end_date)
+
+    df = pd.read_sql_query(query_obj.with_labels().statement, query_obj.session.bind)
+    markers_in_area_list = df.to_dict(orient='records')
+    response = Response(json.dumps(markers_in_area_list, default=str), mimetype="application/json")
+    response.headers.add('Access-Control-Allow-Origin', '*')
```
Comment (atalyaalon) [inline anyway/flask_app.py]: no need for such header here

File: anyway/flask_app.py [hunk 4]
```diff
@@ -1311,6 +1312,29 @@ def get_dict_file(directory):
         yield name, df
 
 
+@app.route("/markers/polygon/", methods=["GET"])
```
Comment (atalyaalon) [inline anyway/flask_app.py]: @erangil  can you add an example of a use in the API?

File: anyway/flask_app.py (hunk 2 above)
Comment (erangil) [reply anyway/flask_app.py]: Done

File: anyway/flask_app.py (hunk 4 above)
Comment (erangil) [reply anyway/flask_app.py]: Done

**Pull request merged**